ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs

```diff
@@ -33,6 +33,9 @@ class ReactInstance : IReactInstance
 
         private bool _initialized;
 
+        public event TransitionToBridgeIdleCallback OnTransitionToBridgeIdleCallback;
```

Comment:
>TransitionToBridgeIdleCallback [](start = 21, length = 30)

Any reason we can't just use `Action` here? Or, switch to the more common usage of `EventHandler<SomeEventArgs>`?

Reply:
Since it's just a binary, what would the additional data be to justify an Action or an EventHandler + new Argument type?

Reply:
@matthargett - I'm fine with just using a non-generic `Action` then so we don't need to invent a new data type, or a non-generic `EventHandler` delegate type.

Reply:
#fixed